vnext/Microsoft.ReactNative/Fabric/Composition/ScrollViewComponentView.cpp

```diff
@@ -726,6 +727,14 @@ void ScrollViewComponentView::updateProps(
 
   // update BaseComponentView props
   base_type::updateProps(props, oldProps);
+
+  // Update the color only after updating the m_props in BaseComponentView
+  // to avoid scrollbarcomponents reading outdated scrollEnabled value.
+  if (!oldProps || oldViewProps.scrollEnabled != newViewProps.scrollEnabled) {
+    m_scrollVisual.ScrollEnabled(newViewProps.scrollEnabled);
+    m_horizontalScrollbarComponent->OnThemeChanged();
+    m_verticalScrollbarComponent->OnThemeChanged();
```

Comment:
If the scrollbarcomponent function is now used for ThemeChange and enabled/disabled change it shouldnt be called OnThemeChanged.  Maybe rename it to UpdateColors or something like that.